R/dfm.R

```diff
@@ -236,18 +236,26 @@ dfm.tokenizedTexts <- function(x,
     if (is.null(names(x))) {
         names(x) <- paste("text", seq_along(x), sep="")
     } 
-    
-    # use tokens_lookup for dictionaries with multi-word values otherwise do this later
-    if (!is.null(dictionary) | !is.null(thesaurus)) {
+
+    # use tokens_lookup for tokens objects
+    if (!is.null(dictionary) || !is.null(thesaurus)) {
         if (!is.null(thesaurus)) dictionary <- dictionary(thesaurus)
-        if (any(stringi::stri_detect_fixed(unlist(dictionary, use.names = FALSE), 
-                                           attr(dictionary, 'concatenator')))) {
-            if (verbose) catm("   ... ")
-            x <- tokens_lookup(x, dictionary,
-                               exclusive = ifelse(!is.null(thesaurus), FALSE, TRUE),
-                               valuetype = valuetype,
-                               verbose = verbose)
-            dictionary <- thesaurus <- NULL
+        if (verbose) catm("   ... ")
+        x <- tokens_lookup(x, dictionary,
+                           exclusive = ifelse(!is.null(thesaurus), FALSE, TRUE),
+                           valuetype = valuetype,
+                           verbose = verbose)
+    }
+    
+    # use tokens_select for tokens objects
+    if (!is.null(c(remove, select))) {
+        if (verbose) catm("   ... ")
```

Comment:
This should precede a message output in `tokens_select()` when `verbose = TRUE`. But in `tokens_select()` there is no output message. Can you please add one that will be similar to https://github.com/kbenoit/quanteda/blob/issue-836/tests/testthat/test-dfm.R#L377-L386, which you can then modify and uncomment.